Binance Detect Moonings.py

```diff
@@ -360,12 +362,16 @@ def sell_coins():
 
     for coin in list(coins_bought):
         # define stop loss and take profit
-        TP = float(coins_bought[coin]['bought_at']) + (float(coins_bought[coin]['bought_at']) * coins_bought[coin]['take_profit']) / 100
-        SL = float(coins_bought[coin]['bought_at']) + (float(coins_bought[coin]['bought_at']) * coins_bought[coin]['stop_loss']) / 100
-
+        BuyPrice = float(coins_bought[coin]['bought_at'])
+        try:
+            TP = BuyPrice + (BuyPrice) * coins_bought[coin]['take_profit']) / 100
+            SL = BuyPrice + (BuyPrice) * coins_bought[coin]['stop_loss']) / 100
```

Comment:
This code doesn't run. Remove the extraneous parenthesis:

```suggestion
            TP = BuyPrice + (BuyPrice) * coins_bought[coin]['take_profit'] / 100
            SL = BuyPrice + (BuyPrice) * coins_bought[coin]['stop_loss'] / 100
```